content/040_observability_monitoring_lab/42_lab1.md

```diff
@@ -0,0 +1,95 @@
+
+1. In the right pane, in the Dashboards dropdown, select **AWS Region Events of Interest**. For more information on the RCE and Dashboards linked, refer to our [Root Cause Explorer](https://help.sumologic.com/Observability_Solution/Root_Cause_Explorer) documentation. 
+
+    ![Region Events of Interest](/images/lab-monitoring/lab1-regionevent-interest.png)
+
+1. To open the Events of Interest (EOI) dashboard in the Root Cause Explorer (RCE), click the three vertical dots on the right of the EoI screen. 
+
+    ![Root Cause Explorer](/images/lab-monitoring/lab1-open-rce.png)
+
+    The EOI opens in RCE.
+
+    ![EOI in RCE](/images/lab-monitoring/lab1-eoi-rce.png)
+
+1. You can see the most recent outage in the EC2 ALB with 5XX error plotted on the screen.
+
+    When you hover over the circles, you can see the EOI stats. In the stats, you can see that one of the reading says y% for x min. It indicates that the value of the metric shows **y** percent drift from the expected value for **x** minutes. The extent of drift from the expected value of a metric are classified as High, Medium or Low. The high intensity EOIs require more attention than others.
+
+    ![EOI Error](/images/lab-monitoring/lab1-eoi-error.png)
+
+1. However, if you hover your mouse slightly above 5XX error, you can see the EC2 CPU Log bottleneck event is plotted slightly earlier, and when you move further up, you’ll notice the DynamoDB throttle event plotted earlier than the CPU Load bottleneck. 
+
+    ![EOI Error](/images/lab-monitoring/lab1-eoi-error.png)
+
+    ![EOI Bottleneck](/images/lab-monitoring/lab1-eoi-bottleneck.png)
+
+1. You can select the time range and the scatter plot will magnify. Use the magnifying glass to zoom out.
+1. When you put these relevant events in timeline perspective, you realise that mostly the DynamoDB got throttled first and rest of the events occurred as the cascading effects of the database throttling.
+
+    ![Find DynamoDB Throttle](/images/lab-monitoring/lab1-find-throttle.png)
+
+1. Click the **DynamoDB** event plot. On the right side of the screen, in the Entity Inspector panel, check the details that the **Summary** tab displays. You can see a spike in the metrics chart plotted below in the Summary tab.
+
+    ![Event Summary](/images/lab-monitoring/lab1-rce-summary.png)
+
+1. Click the **Entities** tab, where you can see the relevant entities and environment parameters listed.
+
+    ![Entities tab](/images/lab-monitoring/lab1-rce-entities.png)
+
+1. Click the **Open In** button in the Entity section, and select **Entity Dashboard**, to open the relevant entity dashboard.
+
+    ![Entity Dashboard](/images/lab-monitoring/lab1-entity-dashboard.png)
+
+1. Open the relevant DynamoDB Events dashboard by clicking the **AWS DynamoDB - Events** option in the dropdown The data in the dashboard is fetched using the relevant metrics.
+
+    ![AWS DynamoDB Events](/images/lab-monitoring/lab1-dynamodb-events.png)
```

Comment:
Only first reference of the service name needs to be "AWS" + Service name. So can remove AWS reference for future references. Pls update rest of the docs with the same.